lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -939,41 +939,53 @@ func (p *ConnPool) GetTaskList(index string, min, max int) ([]string, error) {
 1. index is the name of the index under which the key needs to be deleted
 2. key is the id of the resource to be deleted under an index
 */
-func (p *ConnPool) Del(index, key string) error {
+func (p *ConnPool) Del(index string, cursor float64, key string) error {
```

Comment:
why cursor value is required to be passed by the caller?

Reply:
addressed